build err

**Describe the bug**
# github.com/segmentio/kafka-go
../../../github.com/segmentio/kafka-go/crc32.go:60:4: undefined: io.StringWriter

-bash-4.2$ go get -u github.com/segmentio/kafka-go
# github.com/segmentio/kafka-go
../../../github.com/segmentio/kafka-go/crc32.go:60:4: undefined: io.StringWriter

--- a/crc32.go
+++ b/crc32.go
@@ -3,7 +3,6 @@ package kafka
 import (
 	"encoding/binary"
 	"hash/crc32"
-	"io"
 )
 
 type crc32Writer struct {
@@ -54,8 +53,3 @@ func (w *crc32Writer) WriteString(s string) (int, error) {
 	w.update([]byte(s))
 	return len(s), nil
 }
-
-var (
-	_ io.Writer       = (*crc32Writer)(nil)
-	_ io.StringWriter = (*crc32Writer)(nil)
-)

